src/theme/index.css

```diff
@@ -66,7 +68,7 @@ body {
 }
 
 /* Footer Toolbox Background Selected */
-._b3894505-formatting-toolbar-item[data-active='true'] {
+._b3894505-formatting-toolbar-item[data-active="true"] {
```

Comment:
Would be good if you could change that back to `'` across the file, as it was before